### PR TITLE
Fix compiler warnings about ambiguous code

### PR DESCRIPTION
**Pull request by fingolfin**
I.e., add braces or parentheses or newlines to make it crystal clear
that the code in question really does what it is supposed to.

### CODE REVIEW

File: functions/Autgrp/isometry.c [hunk 1]
```diff
@@ -458,7 +458,8 @@ isometry (matrix_TYP **F1, matrix_TYP **F2, int Fanz, matrix_TYP *SV1, matrix_TY
 	}
 /* V.prime is a prime p, such that the entries of the short vectors remain 
    unchanged under reduction mod p in symmetric form, i.e. -p/2 < x <= p/2 */
-	for (V.prime = 2*max + 1; isprime(V.prime) == 0; ++V.prime);
+	for (V.prime = 2*max + 1; isprime(V.prime) == 0; ++V.prime)
+	    ;
```
Comment (DominikBernhardt) [inline functions/Autgrp/isometry.c]: Why did you put the semicolon into a new line here?

File: functions/Name/compute_q_matrix.c [hunk 2]
```diff
@@ -18,13 +18,16 @@ static int cmp_mat_lex (matrix_TYP *A, matrix_TYP *B)
   int i, j;
 
   
-      for (i=0; i<A->rows; i++)
-	for (j=0; j<A->cols; j++)
-	  if (A->array.SZ[i][j] != B->array.SZ[i][j])
+      for (i=0; i<A->rows; i++) {
+	for (j=0; j<A->cols; j++) {
```
Comment (DominikBernhardt) [inline functions/Name/compute_q_matrix.c]: Should we maybe indent the second `for` here as it is a nested `for` loop? Might make it easier to read the code.

File: functions/Autgrp/isometry.c (hunk 1 above)
Comment (fingolfin) [reply functions/Autgrp/isometry.c]: To indicate to the compiler that yes, an empty loop body is what we want; we didn't just by accident put a semicolon after the `for`.

An alternative would be to replace the semicolon by an empty block `{}`.

File: functions/Name/compute_q_matrix.c (hunk 2 above)
Comment (fingolfin) [reply functions/Name/compute_q_matrix.c]: It is actually indented, but this code is using tabs, so the indentation is not properly shown here in the web UI; but it is there. Of course this is one of the things that using `clang-format` would take care of.

File: functions/Autgrp/isometry.c (hunk 1 above)
Comment (DominikBernhardt) [reply functions/Autgrp/isometry.c]: Ah ok, that's fine by me then.

File: functions/Name/compute_q_matrix.c (hunk 2 above)
Comment (DominikBernhardt) [reply functions/Name/compute_q_matrix.c]: I think we'll merge this now and potentially work on the code formatting in a seperate PR.

**Pull request merged**